package-lock.json

```diff
@@ -217,6 +217,12 @@
       "integrity": "sha512-dInnr7nSPsofnLggOf70xvsInUjf3tRrE8XmxsioXALWQHkwEWi7RhTBCa9mYmiUDHMtuanSDN/JOW187ChIhw==",
       "dev": true
```

Comment:
Revert this file!

Reply:
Will revert after all the review is done.